ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/statementofmeans/CCDBankAccount.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.cmc.ccd.domain.statementofmeans;
+
+import lombok.Builder;
+import lombok.Value;
+import uk.gov.hmcts.cmc.ccd.domain.CCDYesNoOption;
+
+import java.math.BigDecimal;
+
+@Value
+@Builder
+public class CCDBankAccount {
+
+    private BankAccountType type;
```

Comment:
I think we should reuse `uk.gov.hmcts.cmc.domain.models.statementofmeans.BankAccount.BankAccountType` instead of creating enum duplicate. That should ease maintenance in the future.

Reply:
The same applies to there enum duplicates.